static/src/inline-svgs/icon/arrow-up.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 18"><path d="M.4 15.3l10.5-8.4 1.1-.9 1.1.9 10.5 8.4-.5.7-11.1-6.3-11.1 6.3-.5-.7z"/></svg>
```

Comment:
Do you not need to remove xmlns from these for IE8?

See @sndrs doc here, https://github.com/guardian/frontend/blob/master/docs/inline-svgs.md